main.js

```diff
@@ -34,3 +34,303 @@ window.addEventListener('click', disapearEcsResizeClickout);
 menuLinks.forEach((link) => {
   link.addEventListener('click', disapear);
 });
+
+// Section 2 & PopUp Window
+
+const cardsContent = [
```

Comment:
Well done 🚀🔥 The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project.